pkg/handlers/handlers_test.go

```diff
@@ -72,85 +67,6 @@ func BenchmarkHandleFile(b *testing.B) {
 	}
 }
 
-func TestHandleFileSkipBinaries(t *testing.T) {
```

Comment:
Is this logic covered elsewhere or is it no longer relevant?

Reply:
This is no longer relevant, although I can modify this test to cover our logic for skipping files listed in our `ignoredExtensions`.